### PR TITLE
Update site in prep for beta

### PR DESCRIPTION
**Pull request by ameyer-pivotal**
- Update homepage
- Add tutorial
- Polish docs (sync with pack readme)

Signed-off-by: Andrew Meyer <ameyer@pivotal.io>

### CODE REVIEW

File: docs/CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-buildpacks.io
```
Comment (ameyer-pivotal) [inline docs/CNAME]: Not sure why this happened. Reverting...

Comment (ameyer-pivotal) [inline docs/CNAME]: Basically, there are top-level directories called `docs` and `resources`. I'm unsure if they're actually needed or just leftover. I simply copied the real `docs` and `resources` dirs that are located in `src` and replaced the outer ones with those too. I'm not sure this is the right thing to do, so some guidance would be helpful.

This deletion is a result of this file not existing in the original dir I copied from...

**Comment by sclevine** [comment]
@ameyer-pivotal I think I fixed it. Just FYI, to build:
```
$ cd src
$ hugo
```

**Comment by jkutner** [comment]
In 8c93090 i made changes to the header and title. Happy to revisit though.

image

**Pull request merged**